content/en/agent/cluster_agent/admission_controller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Datadog Admission Controller
+kind: documentation
+further_reading:
+- link: "/agent/cluster_agent/clusterchecks/"
+  tag: "Documentation"
+  text: "Running Cluster Checks with Autodiscovery"
+- link: "/agent/cluster_agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting the Datadog Cluster Agent"
+---
+
+## Overview
+The Datadog admission controller is a component of the Datadog Cluster Agent. The main benefit of the admission controller is to simplify the user's application pod configuration. For that, It has two main functionalities:
+
+- Inject environment variables (`DD_AGENT_HOST` and `DD_ENTITY_ID`) to configure DogStatsD and APM tracer libraries into the user's app containers.
+- Inject Datadog standard tags (`env`, `service`, `version`) from app labels into the container environment variables.
+
+## Requirements
+
+- Datadog Cluster Agent 1.7.0
+
+## Configuration
+
+### Helm chart
+
+To enable the admission controller for Helm chart, set the parameter `clusterAgent.admissionController.enabled` to `true`:
+
+{{< code-block lang="yaml" filename="values.yaml" disable_copy="true" >}}
+[...]
+ clusterAgent:
+[...]
+  ## @param admissionController - object - required
+  ## Enable the admissionController to automatically inject APM and
+  ## DogStatsD config and standard tags (env, service, version) into
+  ## your pods
+  #
+  admissionController:
+    enabled: true
+
+    ## @param mutateUnlabelled - boolean - optional
+    ## Enable injecting config without having the pod label:
+    ## admission.datadoghq.com/enabled="true"
+    #
+    mutateUnlabelled: false
+[...]
+{{< /code-block >}}
+
+### Datadog operator
+
+To enable the admission controller for the Datadog operator, set the parameter `clusterAgent.config.admissionController.enabled` to `true` in the custom resource:
+
+```yaml
+[...]
+ clusterAgent:
+[...]
+    config:
+      admissionController:
+        enabled: true
+        mutateUnlabelled: false
+[...]
+```
+
+### APM and DogStatsD
+
+Inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` to configure DogstatsD clients and APM tracers automatically by using one of the following:
```

Comment:
Maybe we should mention that by using this admission controller, users don't need to configure the application pods using downward API (step 2 in https://docs.datadoghq.com/agent/kubernetes/apm/?tab=helm)